### PR TITLE
[2201.10.0] Add LanguageServer features to release note

### PR DESCRIPTION
**Pull request by LakshanWeerasinghe**
## Purpose
$subject

### CODE REVIEW

File: downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md [hunk 1]
```diff
@@ -71,12 +71,18 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 10 (2201.10.0)
 
 #### Language Server
 
+- Introduced the `Convert to configurable` code action to convert a module level variable into a configurable variable.
```
Comment (nipunayf) [inline downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md:74]:
```suggestion
- Introduced the `Convert to configurable` code action to convert a module-level variable into a configurable variable.
```

File: downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md [hunk 2]
```diff
@@ -71,12 +71,18 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 10 (2201.10.0)
...
+- Introduced the `Extract to configurable` code action to extract expressions or function arguments into configurable variables.
+- Introduced the `Add to Config.toml` code action to add configurable variables in to the Config.toml file.
```
Comment (nipunayf) [inline downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md:76]:
```suggestion
- Introduced the `Add to Config.toml` code action to add configurable variables into the `Config.toml` file.
```

File: downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md [hunk 3]
```diff
@@ -71,12 +71,18 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 10 (2201.10.0)
...
+
 #### CLI
 
 #### OpenAPI tool
 
 ### Improvements
 
+- Added navigation and reference finding support for init functions.
```
Comment (nipunayf) [inline downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md:84]:
```suggestion
- Added navigation and reference-finding support for `init` functions.
```

File: downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md (hunk 2 above)
Comment (MaryamZi) [reply downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md:76]: Nipuna's suggestion, but replacing "into" with just "to".

File: downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md (hunk 3 above)
Comment (MaryamZi) [reply downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md:84]: Object init, right? Let's include that.

**Comment by sm1990** [comment]
LGTM

File: downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md [hunk 4]
```diff
@@ -71,12 +71,18 @@ To view bug fixes, see the [GitHub milestone for Swan Lake Update 10 (2201.10.0)
...
+- Introduced the `Extract to configurable` code action to extract expressions or function arguments into configurable variables.
+- Introduced the `Add to Config.toml` code action to add configurable variables to the `Config.toml` file.
+
 #### CLI
 
 #### OpenAPI tool
 
 ### Improvements
 
+- Added navigation and reference-finding support for object `init` functions.
```
Comment (MaryamZi) [inline downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md:84]: Shouldn't this come under "Language Server"?

Comment (LakshanWeerasinghe) [reply downloads/swan-lake-release-notes/swan-lake-2201.10.0/RELEASE_NOTE.md]: updated

**Review by MaryamZi — APPROVED** [review]

**Pull request merged**